cmd/state-svc/main.go

```diff
@@ -38,26 +38,40 @@ const (
 func main() {
 	var exitCode int
 
+	var cfg *config.Instance
 	defer func() {
 		if panics.HandlePanics(recover(), debug.Stack()) {
 			exitCode = 1
 		}
+
+		if err := cfg.Close(); err != nil {
+			logging.Error("Failed to close config after exiting systray: %w", err)
+		}
```

Comment:
Can't we put this in the `WaitForEvents` call below?

Reply:
Unfortunately no, the `WaitForEvents` only takes functions that do not return errors